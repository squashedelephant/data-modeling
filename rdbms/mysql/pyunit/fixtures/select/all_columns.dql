SELECT * FROM account_history;
