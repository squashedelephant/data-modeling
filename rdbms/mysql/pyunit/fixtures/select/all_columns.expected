1L
1L
2L
PLACEHOLDER
PLACEHOLDER
100.00
2L
1L
4L
PLACEHOLDER
PLACEHOLDER
40.00
